profiles/fscloud/variables.tf

```diff
@@ -0,0 +1,125 @@
+variable "plan" {
+  type        = string
+  description = "Plan for the event stream instance. lite, standard or enterprise-3nodes-2tb"
+  default     = "enterprise-3nodes-2tb"
+}
+
+variable "resource_group_id" {
+  description = "ID of resource group to use when creating the event stream instance"
+  type        = string
+}
+
+variable "tags" {
+  type        = list(string)
+  description = "List of tags associated with the Event Steams instance"
+  default     = []
+}
+
+variable "es_name" {
+  description = "Name of the event streams instance"
+  type        = string
+}
+
+variable "schemas" {
+  type = list(object(
+    {
+      schema_id = string
+      schema = object({
+        type = string
+        name = string
+      })
+    }
+  ))
+  description = "The list of schema object which contains schema id and format of the schema"
+  default = [{
```

Comment:
The fscloud module should not have any schemas set by default. So default should be set to `[]`

Reply:
Done